ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/cucushift-installer-rehearse-vsphere-upi-workflow.yaml

```diff
@@ -10,6 +10,7 @@ workflow:
     - ref: upi-conf-vsphere-ova
     - ref: upi-install-vsphere
     - ref: ipi-install-times-collection
+    - ref: enable-qe-catalogsource
```

Comment:
I think we can replace all the pre steps with `cucushift-installer-rehearse-vsphere-upi-provision` chain, and all the step with `cucushift-installer-rehearse-vsphere-upi-deprovision` chain.